mne/report.py

```diff
@@ -550,6 +550,14 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
             {{img}}
         </div>
     {{endif}}
+    {{if textbox is not None}}
```

Comment:
don't you need a default value? otherwise it will break everything else where `textbox` is not used in `substitute`.